addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -214,10 +214,10 @@ def _compute_l10n_latam_available_document_types(self):
 
     @api.depends('l10n_latam_available_document_type_ids', 'debit_origin_id')
     def _compute_l10n_latam_document_type(self):
-        debit_note = self.debit_origin_id
         for rec in self.filtered(lambda x: x.state == 'draft'):
+            debit_note = rec.debit_origin_id
```

Comment:
That looks like a probable pre-existing bug, shouldn't it be moved out of this PR and to a fix to wherever this was introduced? Or was it previously impossible to trigger by construction?

Reply:
It is.